src/ssSnow.lua

```diff
@@ -0,0 +1,128 @@
+---------------------------------------------------------------------------------------------------------
+-- SCRIPT TO ADD PHYSICAL SNOW LAYERS
+---------------------------------------------------------------------------------------------------------
+-- Purpose:  to create plowable snow on the ground
+-- Authors:  Authors:  mrbear
+--
+
+ssSnow = {};
+local modDir = g_currentModDirectory;
+
+
+function ssSnow.preSetup()
+end
+
+function ssSnow.setup()
+    addModEventListener(ssSnow)
+end
+
+function ssSnow:loadMap(name)
+
+
+    -- Register Snow as a fill and Tip type
+
+    FillUtil.registerFillType("snow",  g_i18n:getText("fillType_snow"), FillUtil.FILLTYPE_CATEGORY_BULK, 0,  false,  "dataS2/menu/hud/fillTypes/hud_fill_straw.png", "dataS2/menu/hud/fillTypes/hud_fill_straw_sml.png", 0.0002* 0.5, math.rad(50));
+    TipUtil.registerDensityMapHeightType(FillUtil.FILLTYPE_SNOW, math.rad(35), 0.8, 0.10, 0.10, 1.20, 3, true, modDir .. "resources/snow_diffuse.dds", modDir .. "resources/snow_normal.dds", modDir .. "resources/snowDistance_diffuse.dds");
+    print("Debug: " .. FillUtil.FILLTYPE_SNOW );
+    -- General initalization
+    
+    g_currentMission.environment:addHourChangeListener(self);
+    if g_currentMission.missionInfo.timeScale > 120 then 
```

Comment:
Should be >

Needed for the high speeds we use for testing, not enough calls to update to Finnish an update in a game hour. Should be used for normal players. Does the entire map at ones, locking the game instead of updating gradually.